<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageCreate</class>
 <widget class="QWidget" name="PageCreate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1198</width>
    <height>455</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="5,1,1">
   <item>
    <widget class="QLabel" name="labelDragImg">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../res/resources.qrc">:/images/images/drag-drop.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelDragText">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;拖放素材文件到此以添加内容或&lt;a href=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#808080;&quot;&gt;浏览&lt;/span&gt;&lt;/a&gt;...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
</string>
     </property>
     <property name="textFormat">
      <enum>Qt::MarkdownText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelSizeHint">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
